nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -128,11 +128,7 @@ let
     name = "initrd-emergency-env";
```

Comment:
general bin or emergency bin?

Reply:
Ah looks like we forgot that one mention ;)